lib/pull.js

```diff
@@ -69,7 +69,9 @@ const fs = require('fs'),
                     let cmd = "[ -f " + sourcePath + " ] && echo 'f' || echo 'nf'";
 
                     session.run(cmd, null, writeIsFile, null, function(){
-                        console.log("Copying data ....");
+                        if(!options.silent) {
```

Comment:
Does this not affect the existing case?
Already relate cases are in qwa, qsdk test case.

Reply:
Not affect it. I've already check it. 
`$ ./ares-pull.js /tmp/testPull.txt .
Copying data ....
Pull: /tmp/testPull.txt -> ./testPull.txt
1 file(s) pulled
138 KB/s (4096 bytes in 0.029s)
ye0607kim@ye0607kim-linux:~/project/github/ares-cli/bin$ ./ares-pull.js /tmp/testPull.txt . -i
Copying data ....
1 file(s) pulled
103 KB/s (4096 bytes in 0.039s)`